app/lib/common/widgets/chat/convo_card.dart

```diff
@@ -37,56 +40,35 @@ class ConvoCard extends ConsumerStatefulWidget {
   });
 
   @override
-  ConsumerState<ConvoCard> createState() => _ConvoCardState();
-}
-
-class _ConvoCardState extends ConsumerState<ConvoCard> {
-  List<Member> activeMembers = [];
-
-  @override
-  void initState() {
-    super.initState();
-    getActiveMembers();
-  }
-
-  @override
-  Widget build(BuildContext context) {
-    String roomId = widget.room.getRoomIdStr();
+  Widget build(BuildContext context, WidgetRef ref) {
+    final titleIsLoading = !ref.watch(roomDisplayNameProvider(roomId)).hasValue;
     final roomAvatarInfo = ref.watch(roomAvatarInfoProvider(roomId));
-    final latestMsg = ref.watch(latestMessageProvider(widget.room));
     // ToDo: UnreadCounter
     return ConvoWithAvatarInfoCard(
+      animation: animation,
       roomId: roomId,
-      showParents: widget.showParents,
-      showSuggestedMark: widget.showSuggestedMark,
+      showParents: showParents,
+      showSuggestedMark: showSuggestedMark,
       avatarInfo: roomAvatarInfo,
-      onTap: widget.onTap,
-      showSelectedIndication: widget.showSelectedIndication,
-      subtitle: latestMsg != null
-          ? _SubtitleWidget(
-              room: widget.room,
-              latestMessage: latestMsg,
-            )
-          : const SizedBox.shrink(),
-      trailing: widget.trailing ?? renderTrailing(context),
+      onTap: onTap,
+      title: titleIsLoading ? Skeletonizer(child: Text(roomId)) : null,
```

Comment:
notice how we are here showing a `Skeletonizer` when the `displayName` isn't available yet. this reduces the times we see `roomId`s shown in the UI for chat altogether.